content/management/configuration-api/v3.2/index.mdx

```diff
@@ -32,6 +32,720 @@ Summing up, if the user token starts with `fra-`, you should add the `X-Region:
+- `new_goal`
+- `unassigned_chats`
+- `unassigned_chat_messages`
+- `ticket`
+- `inactive_notifications`
+- `mute_all_sounds`
+- `repeat_sound_notifications`
+- `ticket_notifications`
+
+__****__ Possible values for the `email_subscriptions` array (value present in array means that notification is turned on):
+- `daily_summary`
+- `weekly_summary`
+
+__*****__ The table below presents the possible values for the `groups[].priority` parameter:
+
+| Possible value |                                                                                                      Notes                                                                                                      |
+|----------------|:---------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------:|
+| `first`        |                         The highest chat routing priority. Agents with the `first` priority get chats before others from the same group, e.g. Bots can get chats before regular Agents.                         |
+| `normal`       | The medium chat routing priority. Agents with the `normal` priority get chats before those with the `last` priority, when there are no Agents with the `first` priority available with free slots in the group. |
+| `last`         |               The lowest chat routing priority. Agents with the `last` priority get chats when there are no Agents with the `first` or `normal` priority available with free slots in the group.                |
+
+</Text>
+
+<Code>
+
+<CodeSample path={'REQUEST'}>
+
+```shell
+curl -X POST \
+  https://api.livechatinc.com/v3.2/configuration/action/create_agent \
+  -H 'Authorization: Bearer <your_access_token>' \
+  -H 'Content-Type: application/json' \
+  -d '{
+        "id": "agent@domain.com",
+        "name": "New Agent"
+        "role": "viceowner",
+        "groups": [
+          {
+            "id": 5,
+            "priority": "first"
+          },
+          {
+            "id": 2,
+            "priority": "last"
+          },
+          {
+            "id": 1,
+            "priority": "normal"
+          }
+        ],
+        "work_scheduler": {
+          "monday": {
+            "start": "8:30",
+            "end": "12:30"
+          },
+          "friday": {
+            "start": "7:30",
+            "end": "21:30"
+          }
+        },
+        "notifications": [
+          "new_visitor",
+          "new_goal",
+          "visitor_is_typing"
+        ],
+        "summaries": [
+          "weekly_summary"
+        ]
+      }'
+```
+
+</CodeSample>
+
+<CodeResponse>
+
+```json
+{
+    "id": "agent@domain.com",
+}
+```
+
+</CodeResponse>
+
+</Code>
+
+</Section>
+
+<Section>
+
+<Text>
+
+### Get Agent
+
+It returns the info about an Agent specified by `id`. There are some agent fields that always gonna be returned, but user can request some additional ones.
```

Comment:
```suggestion
It returns the info about an Agent specified by `id`.
```

Reply:
I don't think it's necessary to mention that users can request additional fields. I think they know that